### PR TITLE
Subaccounts for load tests

### PR DESCRIPTION
**Pull request by volovyks**
@ChaoticTempest I was not able to use Workspaces `create_tla` on testnet. But the subaccount approach works just fine.

### CODE REVIEW

**Comment by ChaoticTempest** [comment]
Did you follow all the rules for creating a Top level account? You could just use `dev_create_account()` to facilitate this without needing to know the rules, but both functions uses the testnet faucet which can still be drained

File: load-tests/src/fastauth/mod.rs [hunk 1]
```diff
@@ -29,14 +30,35 @@ use near_primitives::{
...
+
+    let worker = near_workspaces::testnet().await.unwrap();
+
+    let root_account = Account::from_secret_key(
+        near_workspaces::types::AccountId::try_from("dev-1660670387515-45063246810397".to_string()).unwrap(),
+        near_workspaces::types::SecretKey::from_str(
+            "ed25519:4hc3qA3nTE8M63DB8jEZx9ZbHVUPdkMjUAoa11m4xtET7F6w4bk51TwQ3RzEcFhBtXvF6NYzFdiJduaGdJUvynAi"
+        ).unwrap(),
+        &worker
+    );
```
Comment (ChaoticTempest) [inline load-tests/src/fastauth/mod.rs:46]: the issue with this is that somebody can easily disrupt the load test by getting this key. We wouldn't notice this either until we run it locally manually. We should be creating a dev account each time and delete the account at the end of the test

Comment (volovyks) [reply load-tests/src/fastauth/mod.rs:46]: Yep, I know. But `dev_create_account` does not work for me. And there is [no Error handling](https://github.com/near/near-workspaces-rs/issues/353).

**Comment by volovyks** [comment]
@ChaoticTempest what kind of rules? Here is what I'm doing:
```rust
let worker = near_workspaces::testnet().await.unwrap();
let w_account = worker.dev_create_account().await.expect("Failed to create account");
tracing::info!("Created worker account: {}", w_account.id());
```
It's not throwing any errors, but the account is never created on testnet.

**Comment by volovyks** [comment]
@ChaoticTempest looks like subaccounts is a way to go for now: https://github.com/near/near-workspaces-rs/issues/354

**Comment by volovyks** [comment]
@ChaoticTempest added delete account step

**Review by ChaoticTempest — APPROVED** [review]

**Review by ppca — APPROVED** [review]

**Pull request merged**